modules/virt-configuring-node-exporter-service.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-node-exporter-service_{context}"]
+= Configuring the node exporter service
+
+The `node-exporter` agent is deployed on every virtual machine in the cluster from which you want to collect metrics. Configure the `node-exporter` agent as a service to expose internal metrics and processes that are associated with virtual machines.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+* Create the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+* Configure the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project by adding `enableUserWorkload` and setting its value to `true`.
+
+.Procedure
+
+. Create the `node-exporter` YAML file. In the following example, the file is called `node-exporter-service.yaml`.
++
+[source,yaml]
+----
+kind: Service
+apiVersion: v1
+metadata:
+  name: node-exporter-service <1>
+  namespace: dynamation <2>
+  labels:
+    servicetype: metrics <3>
+spec:
+  ports:
+    - name: exmet
+      protocol: TCP
+      port: 9100 <4>
+      targetPort: 9100 <5>
+  type: ClusterIP
+  selector:
+    monitor: metrics <6>
+----
+<1> The `node-exporter` service that exposes the metrics from the virtual machines.
+<2> Namespace where the service is created.
+<3> Label for the selector that applies to the `node-exporter` service.
+<4> TCP port number of the virtual machine that is configured with the `servicetype` label.
```

Comment:
sorry I missed this earlier, but port: <4> is the port the service listens on.
targetport: <5> is the port of the pod/VM.

Also, the virtual machine will be configured with the "monitor" label and not the servicetype label.
The servicetype label is used by the ServiceMonitor.

in short, the ports in <4> and <5> are backwards and the labels in <4> and <6> are backwards.

Reply:
Revamped the content. Please review.